tests/functional/monitoring/prometheus/metrics/test_ocs_exporter_go_ceph_migration.py

```
@@ -0,0 +1,413 @@
+        len(samples),
+    )
+
+
+@runs_on_provider
+@pytest.mark.polarion_id("OCS-6006")
+@blue_squad
+@tier2
+@skipif_external_mode
+@skipif_mcg_only
+@skipif_ms_consumer
+def test_ceph_blocklist_operations_via_go_ceph(metric_families, ceph_toolbox_pod):
```

Ruff (0.15.21)

[warning] Unused function argument: `ceph_toolbox_pod`

(ARG001)